packages/theme-chalk/src/select-dropdown-v2.scss

```diff
@@ -14,12 +14,29 @@
   }
 
   @include e(option-item) {
+    @include when(selected) {
+      &:not(.is-multiple).is-disabled {
+        color: getCssVar('text-color-placeholder');
+        &::after {
+          background-color: getCssVar('text-color-placeholder');
+        }
+      }
+    }
+
     &:hover {
       &:not(.hover) {
         background-color: transparent;
       }
     }
   }
+
+  @include when(multiple) {
+    .#{$namespace}-select-dropdown__option-item.is-disabled {
```

Comment:
One suggestion, I think it's better to write `is-disabled` in next line line.

We can see a little bit more clearly what its states are.

Reply:
I think it's acceptable.